util/s3.py

```diff
@@ -1,189 +0,0 @@
-from nose.tools import set_trace
```

Comment:
This file was an outdated duplicate for core/s3.py. Or, now that I think about it, maybe this file is why the import in cdn was working. 😱